test/functional/net.py

```diff
@@ -32,6 +32,42 @@ def setup_network(self):
         self.sync_all()
 
     def run_test(self):
+        self._test_connection_count()
+        self._test_getnettotals()
+        self._test_getnetworkinginfo()
+        self._test_getaddednodeinfo()
+
+    def _test_connection_count(self):
+        assert_equal(self.nodes[0].getconnectioncount(), self.num_nodes)
+
+    def _test_getnettotals(self):
+        # check that getnettotals totalbytesrecv and totalbytessent
+        # are consistent with getpeerinfo
+        peer_info = self.nodes[0].getpeerinfo()
+        assert_equal(len(peer_info), self.num_nodes)
+        net_totals = self.nodes[0].getnettotals()
+        assert_equal(sum([peer['bytesrecv'] for peer in peer_info]),
+                     net_totals['totalbytesrecv'])
+        assert_equal(sum([peer['bytessent'] for peer in peer_info]),
+                     net_totals['totalbytessent'])
+        # test getnettotals and getpeerinfo by doing a ping
+        # the bytes sent/received should change
+        # note ping and pong are 32 bytes each
+        self.nodes[0].ping()
+        time.sleep(0.1)
+        peer_info_after_ping = self.nodes[0].getpeerinfo()
+        net_totals_after_ping = self.nodes[0].getnettotals()
+        for before, after in zip(peer_info, peer_info_after_ping):
+            assert_equal(before['bytesrecv_per_msg']['pong'] + 32,
+                         after['bytesrecv_per_msg']['pong'])
```

Comment:
micro-nit: I think because of indentation these are clearer if you don't wrap them over two lines. That's just my personal preference though.